frontend/src/components/thanks/index.tsx

```diff
@@ -41,17 +41,11 @@ class Thanks extends React.Component<Props, State> {
         this.state = {};
     }
 
-    handleClick = () => {
-        const { history } = this.props;
-        history.push('/');
-    };
-
     render() {
         return (
             <Layout>
                 <ThanksContainer>
                     <h2>Takk fyfir að taka þátt!</h2>
```

Comment:
```suggestion
                    <h2>Takk fyrir að taka þátt!</h2>
```
Not your spelling mistake but it'd be embarrassing if the podcasters notice

Reply:
Hahaha, yeah!